google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuerySnapshot.java

```diff
@@ -49,7 +55,95 @@ public Timestamp getReadTime() {
 
   /** Returns the number of documents in the result set of the underlying query. */
   public long getCount() {
```

Comment:
We should add the overload `public long get(@Nonnull AggregateField.CountAggregateField countAggregateField)` in addition to this `getCount()` method.

Reply:
Leaving it in.